From b657f14cfeaa566b1ac036a2f9962cb21d03ae60 Mon Sep 17 00:00:00 2001
From: Preston Petrie <preston159@users.noreply.github.com>
Date: Wed, 18 Jun 2025 16:58:20 -0500
Subject: [PATCH] Reapply "revertme: Restore original stores in place of Bazaar
 for stable release until testing completed."

This reverts commit 3d70486f3f372d6ad4e926cdefe33b71e4f8fe53.
---
 Containerfile                                                   | 2 --
 .../etc/dconf/db/distro.d/00-bazzite-desktop-silverblue-global  | 2 +-
 .../zz0-01-bazzite-desktop-silverblue-dash.gschema.override     | 2 +-
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/Containerfile b/Containerfile
index 88b55673..c8a504c2 100644
--- a/Containerfile
+++ b/Containerfile
@@ -240,7 +240,6 @@ RUN --mount=type=cache,dst=/var/cache \
     --mount=type=bind,from=ctx,source=/,target=/ctx \
     --mount=type=tmpfs,dst=/tmp \
     dnf5 -y install \
-        bazaar \
         iwd \
         twitter-twemoji-fonts \
         google-noto-sans-cjk-fonts \
@@ -490,7 +489,6 @@ RUN --mount=type=cache,dst=/var/cache \
             openssh-askpass \
             firewall-config && \
         dnf5 -y remove \
-            gnome-software \
             gnome-classic-session \
             gnome-tour \
             gnome-extensions-app \
diff --git a/system_files/desktop/silverblue/etc/dconf/db/distro.d/00-bazzite-desktop-silverblue-global b/system_files/desktop/silverblue/etc/dconf/db/distro.d/00-bazzite-desktop-silverblue-global
index 9e069c9b..399d345e 100644
--- a/system_files/desktop/silverblue/etc/dconf/db/distro.d/00-bazzite-desktop-silverblue-global
+++ b/system_files/desktop/silverblue/etc/dconf/db/distro.d/00-bazzite-desktop-silverblue-global
@@ -10,7 +10,7 @@ hide-icon-shadow=false
 menu-button-terminal='ptyxis --new-window'
 menu-button-system-monitor='/usr/bin/missioncenter-helper'
 menu-button-extensions-app='com.mattjakeman.ExtensionManager.desktop'
-menu-button-software-center='/usr/bin/bazaar window --auto-service'
+menu-button-software-center='gnome-software'
 show-activities-button=true
 show-lockscreen=false
 show-power-option=false
diff --git a/system_files/desktop/silverblue/usr/share/glib-2.0/schemas/zz0-01-bazzite-desktop-silverblue-dash.gschema.override b/system_files/desktop/silverblue/usr/share/glib-2.0/schemas/zz0-01-bazzite-desktop-silverblue-dash.gschema.override
index 0773a60c..12c2cacb 100644
--- a/system_files/desktop/silverblue/usr/share/glib-2.0/schemas/zz0-01-bazzite-desktop-silverblue-dash.gschema.override
+++ b/system_files/desktop/silverblue/usr/share/glib-2.0/schemas/zz0-01-bazzite-desktop-silverblue-dash.gschema.override
@@ -1,2 +1,2 @@
 [org.gnome.shell] 
-favorite-apps=['org.mozilla.firefox.desktop', 'steam.desktop', 'net.lutris.Lutris.desktop', 'io.github.kolunmi.bazaar.desktop', 'org.gnome.Ptyxis.desktop', 'org.gnome.Nautilus.desktop']
+favorite-apps=['org.mozilla.firefox.desktop', 'steam.desktop', 'net.lutris.Lutris.desktop', 'org.gnome.Software.desktop', 'org.gnome.Ptyxis.desktop', 'org.gnome.Nautilus.desktop']
-- 
2.49.0

